.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -36,5 +37,5 @@ https://github.com/zfsonlinux/zfs/wiki/Buildbot-Options
 - [ ] I have updated the documentation accordingly.
 - [ ] I have read the [**contributing** document](https://github.com/zfsonlinux/zfs/blob/master/.github/CONTRIBUTING.md).
 - [ ] I have added [tests](https://github.com/zfsonlinux/zfs/tree/master/tests) to cover my changes.
-- [ ] All new and existing tests passed.
+- [ ] All new and existing tests passed _(if not, or not sure, please submit this as a draft-PR instead)_.
```

Comment:
@behlendorf we could make this something like: 
`(Please think about using the draft PR feature if untested)`

Reply:
Done it slightly friendlier and less focussed on the tests.
(I think the relationship with the tests is pretty obvious by the location already)

Reply:
I still think that's too strong. The tests are the tests, we don't need to use draft status to duplicate the test status. I think the metric for "draft" (aka "WIP") is whether _the submitter_ thinks it is ready to merge. If we want to be more explicit, that could be a new option:

```
- [ ] I believe this PR is ready to merge. _(If not, or not sure, please mark this as a draft PR.)_
```